docs/src/main/sphinx/release/release-379.md

```diff
@@ -32,6 +32,7 @@
 * Improve compatibility when Glue storage properties are used. ({issue}`12164`)
 * Prevent data loss when queries modify a table concurrently when Glue catalog
   is used. ({issue}`11713`)
+* Enable commit retries when conflicts occur writing a transaction to a Hive Metastore.
```

Comment:
Needs issue link. Also .. do we need to document this as well?

Reply:
Added a write up in https://github.com/trinodb/trino/issues/12419

And no, I don't think we need to document it. Retries are transparent to the user and don't have configuration toggles or anything like that